cross/cffi/Makefile

```diff
@@ -13,14 +13,15 @@ LICENSE  = MIT
 
 PLIST_TRANSFORM = sed -e 's%@PYTHON_SITE_PACKAGES@%$(PYTHON_LIB_DIR)/site-packages%' -e 's%@PYTHON_VERSION@%$(SPK_SHORT_VERS)%'
 
-POST_INSTALL_TARGET = install_into_cross_env
+PRE_INSTALL_TARGET = install_into_cross_env
 
 include ../../mk/spksrc.python-module.mk
 
 .PHONY: install_into_cross_env
-install_into_cross_env:
-ifneq ($(CROSSENV),)
-	@. $(CROSSENV) && \
-		build-pip install "$(PKG_NAME)==$(PKG_VERS)" && \
-		pip install "$(PKG_NAME)==$(PKG_VERS)"
+install_into_cross_env: 
```

Comment:
Suggest renaming it into `pre_install_into_cross_env`

Reply:
Done.